doc/development/guide/logging.rst

```diff
@@ -29,8 +29,21 @@ To add logging support to components of PennyLane, we must define a module logge
 
 which will be used within the given module, and track directories,
 filenames and function names, as we have defined the appropriate types
-within the formatter configuration (see :class:`pennylane.logging.DefaultFormatter`). With the logger defined, we can selectively add to the logger by if-else statements, which compare the given module’s log-level to any log record
-message it receives. This step is not necessary, as the message will
+within the formatter configuration (see :class:`pennylane.logging.DefaultFormatter`). With the logger defined, we can selectively add to the logger via two methods: 
+
+i) using decorators on the required functions and methods in a given module:
```

Comment:
These points as well as the code blocks within need to be indented to render correctly.

Reply:
Good call